CMakeLists.txt

```diff
@@ -135,7 +141,8 @@ endif()
 
 if (NOT BUILD_SHARED_LIBS OR PUGIXML_BUILD_SHARED_AND_STATIC_LIBS)
   add_library(pugixml-static STATIC
```

Comment:
Why did this change?

Reply:
Is there another way to include the headers in the built library?

Reply:
`target_include_directories` could be used instead... There isn't much of a difference, if any.

Reply:
Static libraries don't include header files, so I'm not sure what this is changing. I'll need to look at how other projects do xcframework setup but it's surprising that any change is necessary here, beyond the change for  framework installation.

Reply:
Also worth noting I guess is that xcframework files are dynamic libraries... I'll need to test this fully to make sure I understand what is going on here.

Reply:
> Also worth noting I guess is that xcframework files are dynamic libraries...

`xcframework` are just a bunch of built `framework`s bundled in a single package using XCode-CLI.


>  I'll need to test this fully to make sure I understand what is going on here.

Of course, I actually appreciate that. These links might be helpful:
https://cmake.org/cmake/help/latest/prop_tgt/FRAMEWORK.html#prop_tgt:FRAMEWORK

https://github.com/AcademySoftwareFoundation/Imath/pull/461#issuecomment-2581658965